policy/README.md

```diff
@@ -31,3 +31,92 @@ The following policy agent configuration directives are currently supported:
 ### `opa` backend configuration
 
 Currently, `opa` backend does not support any configuration.
+
+## Policy Identification
+
+There are three different ways of identifying a policy:
+
+### appraisal policy ID
+
+This is an identifier used in the attestation result as defined by [EAR
+RFC](https://www.rfc-editor.org/rfc/internet-drafts/draft-fv-rats-ear-00.html#name-ear-appraisal-claims).
+Note that in this case, "policy" is used in the sense of the [RATS
+architecture](https://datatracker.ietf.org/doc/html/draft-ietf-rats-architecture-05)
```

Comment:
* s/.../in the sense of "Appraisal Policy for Evidence" as per RATS architecture/
* for the link use https://www.rfc-editor.org/rfc/rfc9334